main/zoe/api/zoe-helpers.md

```diff
@@ -178,7 +179,7 @@ const { swap } = makeZoeHelpers(zoe);
   return zcf.makeInvitation(hook);
 ```
 
-## zoeHelper.inviteAnOffer(options)
+## zoeHelpers.inviteAnOffer(options)
 - `options` `{offerHook, customProperties, expected}` Optional name 
   properties of the 
```

Comment:
This is going away, right? If it's not, we should fix this dangling description

Reply:
Yeah, I'll fix it, even though it's deprecated